src/plugins/grass/modules/r.geomorphon.qgm

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE qgisgrassmodule SYSTEM "http://mrcc.com/qgisgrassmodule.dtd">
+
+<qgisgrassmodule label="Calculates geomorphons (terrain forms) and associated geometry using machine vision approach" module="r.geomorphon">
+	<option key="elevation" answer="" hidden="no"/>
```

Comment:
are there much more parameters that that here exposed?

Reply:
Those are the main parameters, enough to make the module usabble and keep its usage simpler. I can think of adding the extra output parameters and the multiresolution marked as advanced.